src/plugins/data-card/components/sort-area.scss

```
@@ -0,0 +1,52 @@
+@import "../../../components/vars.sass";
+
+$sort-view-card-width: 177px;
+$sort-view-column-width: $sort-view-card-width + 10px;
+$sort-view-font-size:10px;
+$cards-per-row: 3; // this may be dynamic in the future
+$sort-view-width: ($sort-view-card-width * $cards-per-row );
+
+.data-card-tool.display-as-sorted {
+  font-size: $sort-view-font-size;
+  .panel {
+    width: ($sort-view-card-width * 3 );
```

```suggestion
    width: $sort-view-width;
```
I think this change was missed before. It should work as intended, right?

---

Oh I did mean to change that. Will fix and push. Thank you!